topics-project/app/(welcome)/(auth)/signup/actions.js

```diff
@@ -38,21 +38,16 @@ export async function signup(formData) {
     },
   };
 
-  try {
-    const { data, error } = await supabase.auth.signUp(data_);
-    if (error) {
-      throw error;
-    }
+  const { data, error } = await supabase.auth.signUp(data_);
+  if (error) {
+    throw error;
```

Comment:
You removed the try/catch block so throwing the error wouldn't do anything? Or at least won't do as intended. What happens if the sign-up has an error

Reply:
It throws it up to page.js where I handle it by displaying it to the user

Reply:
ahh okay so the try/catch is in the page.js and so if it throws the error, that will catch it

Reply:
LGTM then